packages/client/mutations/EndRetrospectiveMutation.ts

```diff
@@ -124,11 +114,6 @@ export const endRetrospectiveTeamOnNext: OnNextHandler<
       const hasTeamHealth = phases.some((phase) => phase.phaseType === 'TEAM_HEALTH')
       const pathname = `/new-summary/${meetingId}`
       const search = new URLSearchParams()
-      const hasSuggestGroups = !!autogroupReflectionGroups?.length
-      if (hasSuggestGroups) {
```

Comment:
I think this was used to track usage of the feature, but we already have the `autogroup` mutation tracking this, so it can be removed